lib/dnsimple/client/templates.rb

```diff
@@ -146,6 +146,28 @@ def delete_template(account_id, template_id, options = {})
       end
       alias delete delete_template
 
+      # Applies a template to the domain.
+      #
+      # @see https://developer.dnsimple.com/v2/templates/domains/#apply
+      #
+      # @example Apply template 5401 for example.com in account 1010:
+      #   client.templates.apply_template(1010, "example.com", 5401)
+      #
+      # @param  [Fixnum] account_id The account ID
+      # @param  [#to_s] domain_id The Domain ID or name
+      # @param  [#to_s] template_id The template ID
+      # @param  [Hash] options
+      # @return [Dnsimple::Response<nil>]
+      #
+      # @raise  [Dnsimple::NotFoundError]
+      # @raise  [Dnsimple::RequestError]
+      def apply_template(account_id, domain_id, template_id, options = {})
```

Comment:
Cross-referencing [this comment](https://github.com/dnsimple/dnsimple-go/pull/39/files#r71845151).